faq-accordion-card-main/src/App.js

```diff
@@ -0,0 +1,42 @@
+import './App.css';
+import FAQ from './FAQ/FAQ.js'
+import questions from './Data/Data'
+import woman from './Data/Woman'
+import box from './Data/Box'
+import wMobile from './Data/WomanMobile'
+
+function App() {
+
+  let content = <div><p>FAQ</p></div>
+  content = questions.map((q, index) => {
+    return(
+    <FAQ
+    key = {index}
+    question = {q.question}
+    answer = {q.answer}
+     />)
+  })
```

Comment:
Use full variable names - `question` instead of `q`. Also don't use array indexes as keys
```
const content = questions.map(question => 
  <Faq key={`${answer}-${question}`} question={question.question} answer={question.answer}/>
)